bin/src/Validator/SpecGenerator/Template/CssRuleset.php

```diff
@@ -48,10 +49,10 @@ class CssRuleset
      */
     public function getId()
     {
+        // @todo Is this necessary? Why not just use the class constant?
         return static::ID;
     }
```

Comment:
I'd like to keep this one in, as everything is accessible via methods right now. However, I'll add an interface for elements that can be ID'd.